src/main/java/sirius/web/dispatch/HelpDispatcher.java

```diff
@@ -70,7 +70,8 @@ public int getPriority() {
 
     @Override
     public DispatchDecision dispatch(WebContext ctx) throws Exception {
-        if (!ctx.getRequestedURI().startsWith(HELP_PREFIX) || !HttpMethod.GET.equals(ctx.getRequest().method())) {
+        if (!(ctx.getRequestedURI().startsWith(HELP_PREFIX + "/") || ctx.getRequestedURI().endsWith(HELP_PREFIX))
```

Comment:
Hmm… `endsWith(PREFIX)` sounds very strange considering the meaning of the words. Also, given your examples, I don't really see the point of this second check. Could you please explain?

Reply:
I think switching the second check to an equals will make things more clear.